lib/membrane_element_rawvideo/parser.ex

```diff
@@ -97,6 +103,17 @@ defmodule Membrane.Element.RawVideo.Parser do
     {:ok, %{state | queue: <<>>}}
   end
 
+  defp bump_timestamp(%{caps: %{framerate: {0, _}}} = state) do
+    state
+  end
+
+  defp bump_timestamp(state) do
+    use Ratio
+    %{timestamp: timestamp, caps: %{framerate: {num, denom}}} = state
```

Comment:
I'd calculate the frame duration when receiving caps, and just add that duration here

Reply:
If you do that in `handle_caps`, it will be recalculated whenever necessary. And you don't have to recalculate that every buffer.